codalab/bundles/make_bundle.py

```diff
@@ -20,6 +20,16 @@ class MakeBundle(DerivedBundle):
             generated=True,
         )
     )
+    METADATA_SPECS.append(  # support `--store` command, set the
```

Comment:
```suggestion
    METADATA_SPECS.append(
```

Remove this comment?